marketplace/commercelayer-sku/MARKETPLACE.md

```diff
@@ -0,0 +1,29 @@
+# Contentful UI Extension
```

Comment:
The Marketplace files don't have a header, the header is already rendered by the doc-app